.github/workflows/gh-pages.yml

```
@@ -54,6 +54,8 @@ jobs:
         run: "npm run test:wpt"
       - name: Expected results
         run: "npm run test:compare"
+      - name: Adding summary
```

Suggestion to reword the step name, as it doesn’t really “add” anything.

```suggestion
      - name: Test Results
```